source/process/additional-asset-guidelines.rst

```diff
@@ -0,0 +1,32 @@
+===================================
+Additional Asset Guidelines
+===================================
+
+Guidelines for Screenshot Development of Mattermost User Interface
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The purpose is to highlight and illustrate the best features of Mattermost user interface.
+
+Instructions:
```

Comment:
Should we break this up into
1) Setup (use desktop app, set your screen size, use a secondary account)
2) Checklist (favorite channels, center pane, ...)